package/yast2-packager.spec

```diff
@@ -62,6 +65,9 @@ Requires:       /usr/bin/md5sum
 # .process agent
 Requires:       yast2-core >= 2.16.35
 
+# Storage (methods to check the devicegraph)
+Requires:       libstorage-ng-ruby
```

Comment:
libstorage is already an object-oriented library which offer objects of classes like `Partition`, `Filesystem` and so on. It also models the relation between those objects, actions to go from A to B, etc.

I don't think one of yast-storage goals should be to wrap all that with another level of indirection just for the shake of preventing other parts of ruby from accessing libstorage directly. The goal is to make a better libstorage that does not need an extra layer to be useful.

On the other hand, even if we would have decided to add that extra level of indirection and wrap every libstorage class with a Y2Storage decorator (I felt tempted once in the past), the strict type checking performed by SWIG and C++ destroys the idea of duck-typing and makes class wrapping a PITA (that's why we use refinements instead of wrapper classes to decorate).